dags/openweather/transformer.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.providers.postgres.operators.postgres import PostgresOperator
+
+
+default_args = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'email': ['airflow@example.com'],
+    'retries': 0,
+    'retry_delay': timedelta(minutes=1)
+}
+
+with DAG(
+    'transformer',
+    default_args=default_args,
+    description='To transform the raw current weather to a modeled dataset',
+    # Runs same time as the fetcher DAG, but we use
```

Comment:
Just realized this comment is irrelevant (I originally was playing with ExternalTaskSensor but forgot to remove this comment)